charts/kube-prometheus-stack/README.md

```diff
@@ -80,6 +80,34 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 39.x to 40.x
+
+This version updates kube-state-metrics to 4.18.0 and prometheus-node-exporter to 4.1.0 which adds Kubernetes recommended labels on all resources.
+
+If you use your own custom [ServiceMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#servicemonitor) or [PodMonitor](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#podmonitor), please ensure to upgrade their `selector` fields accordingly to the new labels.
+
+Starting from prometheus-node-exporter version 4.0.0, the `node exporter` chart is using the [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). Therefore you have to delete the daemonset before you upgrade.
+
+```console
+kubectl delete daemonset -l app=prometheus-node-exporter
+helm upgrade -i prometheus-node-exporter prometheus-community/prometheus-node-exporter
```

Comment:
you can't copy this 1 to 1. 
please alos test it!
we don't want to install prometheus-node-exporter standalone, which would happen with this command.

Reply:
I'll spin up a kind cluster and give it a go as it's not really clear what are the upgrade issue.